examples/foreach/foreach.description

```diff
@@ -1,2 +1,2 @@
-// The 'Foreach' statement is a looping construct that traverses through the items of a collection of data such as
+// The `foreach` statement is a looping construct that traverses through the items of a collection of data such as
 // arrays, maps, JSON, XML and table.
```

Comment:
```suggestion
// arrays, maps, JSON, XML, and table.
```

We need to add a comma after the item before "and'in a list. This is called the "Oxford Comma".